artifacts/deploy/karmada-etcd.yaml

```diff
@@ -35,17 +35,35 @@ spec:
         - name: etcd
           image: registry.k8s.io/etcd:3.5.13-0
           imagePullPolicy: IfNotPresent
-          livenessProbe:
-            exec:
-              command:
-                - /bin/sh
-                - -ec
-                - 'etcdctl get /registry --prefix --keys-only  --endpoints https://127.0.0.1:2379  --cacert /etc/karmada/pki/etcd-ca.crt --cert /etc/karmada/pki/etcd-server.crt --key /etc/karmada/pki/etcd-server.key'
+          startupProbe:
+            failureThreshold: 24
+            httpGet:
+              host: 127.0.0.1
+              path: /readyz
+              port: 2381
```

Comment:
The port should be 2379, as our `listen-client-urls` is set to `https://0.0.0.0:2379`.

Reply:
```shell
    Liveness:     http-get http://127.0.0.1:2381/livez delay=10s timeout=15s period=10s #success=1 #failure=8
    Readiness:    http-get http://127.0.0.1:2381/readyz delay=0s timeout=15s period=1s #success=1 #failure=3
    Startup:      http-get http://127.0.0.1:2381/readyz delay=10s timeout=15s period=10s #success=1 #failure=24
```

Reply:
kubeadm have the arg of `--listen-metrics-urls=http://127.0.0.1:2381`, so this is 2381, and karmada have not set it.